plugins/backend/backstage-plugin-argo-cd-backend/src/service/router.ts

```diff
@@ -368,11 +368,14 @@ export function createRouter({
     async (request, response) => {
```

Comment:
This looks like it is still leaving the old endpoint in place and adding an additional query param to introduce new functionality. Because of that this looks like that this shouldn't be a major bump.

Am I looking at the wrong thing here? 
Are we able to make these changes without introducing breakages?

Reply:
correct, the query parameter alone does not introduce a breaking change, however, we did add breaking changes to this PR when changing the response object. The change is explained in the changeset file.  

It would be more accurate to say that this pr introduces a new feature of terminating operations, AND cleans up the delete argo application and project process to more accurately explain the results of the delete calls we are making. Before this, there was poor error handling.

Reply:
Cool. Is there a corresponding modification to the frontend portion of this plugin that is needed for it to work with this newer response shape?